examples/text_to_image/train_text_to_image_lora_sdxl.py

```diff
@@ -431,6 +431,16 @@ def parse_args(input_args=None):
         help="debug loss for each image, if filenames are awailable in the dataset",
     )
 
+    parser.add_argument(
+        "--shuffle_tags",
+        action="store_true",
+        help="shuffle tags randomly")
```

Comment:
it seems like 'make style' didn't work, as the ) is on the same line as help

can you rephrase the help statements to have Capitalised initial letter, and a bit more explanation about when shuffle tags might be used?